src/plugins/dbg/quictypes.h

```diff
@@ -1429,6 +1458,21 @@ struct Registration : Struct {
     String GetAppName() {
         return String(AddrOf("AppName"));
     }
+
+    PSTR GetWorkersState() {
+        auto Workers = GetWorkerPool();
+        UCHAR WorkerCount = Workers.WorkerCount();
+        bool HasQueuedWorker = false;
+        for (UCHAR i = 0; i < WorkerCount; i++) {
+            if (Workers.GetWorker(i).IsActive()) {
+                return "ACTIVE";
+            }
+            if (Workers.GetWorker(i).HasWorkQueue()) {
+                HasQueuedWorker = true;
+            }
+        }
+        return HasQueuedWorker ? "QUEUED" : "  IDLE";
```

Comment:
Space? "  IDLE"

Reply:
I have the spacing so all the strings are the same length, to make the formatting in the output nicer.